### PR TITLE
Copy redhat_packages from rho to sonar

### LINKED ISSUE #359: Merge redhat_packages fingerprints from rho to sonar
Part of #307 .

### PR DESCRIPTION
**Pull request by ilyavw**
Cut out quite a few tasks that didn't actually do anything, so please double check that all the intended functionality is still here.

Closes #359

### CODE REVIEW

File: roles/redhat_packages/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+
+- name: gather redhat-packages fact data
+  raw: rpm -qa --qf "%{NAME}|%{VERSION}|%{RELEASE}|%{INSTALLTIME}|%{VENDOR}|%{BUILDTIME}|%{BUILDHOST}|%{SOURCERPM}|%{LICENSE}|%{PACKAGER}|%{INSTALLTIME:date}|%{BUILDTIME:date}|%{DSAHEADER:pgpsig}|%{RSAHEADER:pgpsig}|%{SIGGPG:pgpsig}|%{SIGPGP:pgpsig}|\n"
+  register: redhat_packages_results
+  ignore_errors: yes
+  when: internal_have_rpm
+
+- name: add redhat-packages.results to dictionary
```
Comment (chambridge) [inline roles/redhat_packages/tasks/main.yml]: clean up task name to reflect more of value extraction.

Comment (ilyavw) [reply roles/redhat_packages/tasks/main.yml]: 👌

**Comment by ilyavw** [comment]
Currently blocked on #389

File: roles/redhat_packages/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+- name: gather the last installed red hat package filtered by gpg keys
+  raw: rpm -qa --qf "%{INSTALLTIME} - %{NAME}|%{VERSION}|%{RELEASE}|%{INSTALLTIME}|%{VENDOR}|%{BUILDTIME}|%{BUILDHOST}|%{SOURCERPM}|%{LICENSE}|%{PACKAGER}|%{INSTALLTIME:date}|%{BUILDTIME:date}|%{DSAHEADER:pgpsig}|%{RSAHEADER:pgpsig}|%{SIGGPG:pgpsig}|%{SIGPGP:pgpsig}|\n" | grep 'Key ID 199e2f91fd431d51\|Key ID 5326810137017186\|Key ID 45689c882fa658e0\|Key ID 219180cddb42a60e\|Key ID 7514f77d8366b0d9\|Key ID 45689c882fa658e0' | sort -nr | head -n 1
+  register: redhat_packages_gpg_last_installed
+  ignore_errors: yes
+  when: internal_have_rpm
+
+- name: set fact of last installed rh package filtered by gpg key
+  set_fact:
+    redhat_packages_gpg_last_installed: "{{ redhat_packages_gpg_last_installed }}"
```
Comment (chambridge) [inline roles/redhat_packages/tasks/main.yml]: Do you not need to extract from `stdout` or `stdout_lines[0]`?  Could you add comment with the example output in the results from a scan?

**Comment by ilyavw** [comment]
Thank you for the feedback above @chambridge . I've run a few scans with rho now to get a good baseline of original facts (should have done that days ago!).

Here is a scan of a rhel7 system via rho 
image

Here is a scan of a centos7 system via rho
image

Here is a scan of the same rhel7 system with quipucords 
```
{
    "name": "redhat_packages_gpg_is_redhat",
    "value": "Y"
},
{
    "name": "redhat_packages_gpg_num_rh_packages",
    "value": "361"
},
{
    "name": "redhat_packages_gpg_num_installed_packages",
    "value": "363"
},
{
    "name": "redhat_packages_gpg_last_installed",
    "value": "net-tools-2.0 Installed:Tue 09 Jan 2018 03:02:38 PM EST"
},
{
    "name": "redhat_packages_gpg_last_built",
    "value": "selinux-policy-targeted-3.13.1 Built:Mon 10 Jul 2017 12:02:54 PM EDT"
}
```
And here is a scan or a centos7 system with quipucords
```
{
    "name": "redhat_packages_gpg_is_redhat",
    "value": "N"
},
{
    "name": "redhat_packages_gpg_num_rh_packages",
    "value": "0"
},
{
    "name": "redhat_packages_gpg_num_installed_packages",
    "value": "332"
}
```

**Pull request merged**